### ISSUE TITLE
MSE loss

### ISSUE THREAD

**Issue by afansi** [jharris1679/transaction_signatures#7]
https://github.com/jharris1679/transaction_signatures/blob/fb73af955dc6fe5a11385e48d6c603d0d81f8cac/tx_model/app/model.py#L175

https://github.com/jharris1679/transaction_signatures/blob/fb73af955dc6fe5a11385e48d6c603d0d81f8cac/tx_model/app/model.py#L170

hi, instead of using nn.CrossEntropy here, 
that is:
`
criterion = nn.MSELoss()
return criterion(output, targets)
`
you can instead use F.cross_entropy, that is
`
F.mse_loss(output, targets)
`

This will avoid overloading your memory for nothing as you instanciate new object at each call.
